vms/platformvm/api/static_service.go

```diff
@@ -51,10 +51,11 @@ type StaticService struct{}
 
 // UTXO is a UTXO on the Platform Chain that exists at the chain's genesis.
 type UTXO struct {
```

Comment:
Suggestion:
- Dont change UTXO
- Send array of DepositIDs (len(depositIDs) == len(UTXOs) in Camino struct